pkgs/top-level/aliases.nix

```diff
@@ -900,6 +900,7 @@ mapAliases {
   kodiPlugins = kodiPackages; # Added 2021-03-09;
   kramdown-rfc2629 = throw "'kramdown-rfc2629' has been renamed to/replaced by 'rubyPackages.kramdown-rfc2629'"; # Converted to throw 2024-10-17
   krb5Full = krb5;
+  kreative-square-fonts = kreative-square;
```

Comment:
Nit: Why not just use a `throw` like the other stuff around here. Something like

```
kreative-square-fonts = throw "'kreative-square-fonts' has been renamed to 'kreative-square'";
```

?